### PR TITLE
Remove comment about empty selectors in the service spec

### PR DESCRIPTION
**Pull request by girishkalele**
As discussed, removing the comment about empty selectors in Service specs.

### CODE REVIEW

File: pkg/api/v1/types.go [hunk 1]
```diff
@@ -2031,7 +2031,6 @@ type ServiceSpec struct {
 
 	// This service will route traffic to pods having labels matching this selector.
 	// Label keys and values that must match in order to receive traffic for this service.
-	// If empty, all pods are selected, if not specified, endpoints must be manually specified.
```
Comment (lavalamp) [inline pkg/api/v1/types.go]: "if not specified, endpoints must be manually specified" <--- don't remove this part.

Comment (lavalamp) [inline pkg/api/v1/types.go]: And maybe add, "and the system will not automatically manage them."

**Comment by girishkalele** [comment]
Looks like the firewall name too long issue.

Failed to create firewall e2e-gce-agent-pr-20160524-4-0-minion-e2e-gce-agent-pr-20160524-4-0-http-alt in kubernetes-jenkins-pull

**Comment by girishkalele** [comment]
@lavalamp Just the old comments change - rebased.

**Comment by lavalamp** [comment]
lgtm

**Comment by girishkalele** [comment]
@k8s-bot test this #ignore

**Comment by girishkalele** [comment]
Not sure why this passes pre-commit
```
pkg/api is out of date. Please run hack/update-generated-protobuf.sh
FAILED   ./hack/../hack/verify-generated-protobuf.sh	23s
Verifying ./hack/../hack/verify-generated-swagger-docs.sh
Go version: go version go1.6.2 linux/amd64
+++ [0628 14:57:49] Building the toolchain targets:
```

**Pull request merged**